ci/jira_query.py

```diff
@@ -24,7 +24,8 @@
 # Expected components from the commit message, and directory in src/, src/client or utils/ is also
 # valid.  We've never checked/enforced these before so there have been a lot of values used in the
 # past.
-VALID_COMPONENTS = ('build', 'ci', 'doc', 'gha', 'il', 'mercury', 'test', 'md', 'agent')
+VALID_COMPONENTS = ('build', 'ci', 'doc', 'gha', 'il', 'mercury', 'test', 'md',
```

Comment:
The jira_query.py script is run from the PR target rather than the PR so it's right to add csum here and it's expected that you still get the warning even with the change.

Reply:
That's good to know, thank you.

Reply:
I've made this change (and others) in https://github.com/daos-stack/daos/pull/10736